packages/backend-function/src/cjs_shim.ts

```diff
@@ -0,0 +1,6 @@
+import { createRequire } from 'node:module';
+import path from 'node:path';
+import url from 'node:url';
+global.require = createRequire(import.meta.url);
+global.__filename = url.fileURLToPath(import.meta.url);
+global.__dirname = path.dirname(__filename);
```

Comment:
`globalThis.___` gave warnings with the message `globalThis is not defined` in the lambda console which may be alarming for users but still works so switched to `global.___` here.